src/libDirectoryService/FinalBlockPreProcessing.cpp

```diff
@@ -1206,8 +1206,8 @@ bool DirectoryService::RunConsensusOnFinalBlockWhenDSBackup() {
     return true;
   }
 
-#ifdef VC_TEST_VC_PRECHECK_2
-  if (m_consensusMyID == 3) {
+#if VC_TEST_VC_PRECHECK_2
+  if (m_consensusMyID == 3 && m_mediator.m_currentEpochNum == 2) {
```

Comment:
What's the meaning of "3" and "2"?

Reply:
this is from previous versions of VCTEST

Reply:
Is it possible to give a meaningful name?